README.md

```diff
@@ -164,13 +164,25 @@ by calling `require`:
 var escomplex = require('escomplex');
 ```
 
-It exports one function,
-called `analyse`:
+It exports two functions,
+the primary interface, called `analyse`:
```

Comment:
I reckon this would read better if we gave each function it's own sub-section containing a full description and all the arguments/examples/etc. Then this sentence could just be wrapped up as `...two functions, called `analyse` and `processResults`.

What do you think?